<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QImageViewer</class>
 <widget class="QMainWindow" name="QImageViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QImageViewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/res/icon.png</normaloff>:/res/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="panelTopLayout"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="imageLayout">
      <item>
       <widget class="QPushButton" name="prevButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="locale">
         <locale language="English" country="Kenya"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/res/prev.png</normaloff>:/res/prev.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </item>
      <item>
       <widget class="QPushButton" name="nextButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="locale">
         <locale language="English" country="Kenya"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/res/next.png</normaloff>:/res/next.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="panelBottomLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="saveAsAction"/>
    <addaction name="separator"/>
    <addaction name="settingsAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="parametersMenu">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="rotateRightAction"/>
    <addaction name="rotateLeftAction"/>
    <addaction name="separator"/>
    <addaction name="flipHorizontalAction"/>
    <addaction name="flipVerticalAction"/>
    <addaction name="separator"/>
    <addaction name="deleteFileAction"/>
    <addaction name="separator"/>
    <addaction name="cropAction"/>
    <addaction name="resizeAction"/>
    <addaction name="resizeitemsAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="previewMenu">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>&amp;Preview</string>
    </property>
    <addaction name="previmageAction"/>
    <addaction name="nextimageAction"/>
    <addaction name="separator"/>
    <addaction name="fullscreenAction"/>
    <addaction name="slideshowAction"/>
    <addaction name="separator"/>
    <addaction name="zoomInAction"/>
    <addaction name="zoomOutAction"/>
    <addaction name="zoomWindowAction"/>
    <addaction name="zoomOriginalAction"/>
    <addaction name="separator"/>
    <addaction name="wallpaperAction"/>
   </widget>
   <widget class="QMenu" name="menuExtern_editors">
    <property name="title">
     <string>&amp;Extern editors</string>
    </property>
    <addaction name="editorsNewAction"/>
    <addaction name="editorsManagerAction"/>
   </widget>
   <widget class="QMenu" name="menuShare">
    <property name="title">
     <string>&amp;Share</string>
    </property>
    <addaction name="shareImageShackAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="parametersMenu"/>
   <addaction name="previewMenu"/>
   <addaction name="menuExtern_editors"/>
   <addaction name="menuShare"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="locale">
    <locale language="English" country="Kenya"/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="openAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/file-open.png</normaloff>:/res/file-open.png</iconset>
   </property>
   <property name="text">
    <string>Open image</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/help.png</normaloff>:/res/help.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/file-save.png</normaloff>:/res/file-save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="rotateRightAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/rotate-right.png</normaloff>:/res/rotate-right.png</iconset>
   </property>
   <property name="text">
    <string>Rotate right</string>
   </property>
  </action>
  <action name="rotateLeftAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/rotate-left.png</normaloff>:/res/rotate-left.png</iconset>
   </property>
   <property name="text">
    <string>Rotate left</string>
   </property>
  </action>
  <action name="deleteFileAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/delete.png</normaloff>:/res/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete file</string>
   </property>
  </action>
  <action name="resizeAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/resize.png</normaloff>:/res/resize.png</iconset>
   </property>
   <property name="text">
    <string>Resize</string>
   </property>
  </action>
  <action name="fullscreenAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/fullscreen.png</normaloff>:/res/fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>Fullscreen mode</string>
   </property>
  </action>
  <action name="cropAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/crop.png</normaloff>:/res/crop.png</iconset>
   </property>
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="wallpaperAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/wallpaper.png</normaloff>:/res/wallpaper.png</iconset>
   </property>
   <property name="text">
    <string>Set as wallpaper</string>
   </property>
  </action>
  <action name="slideshowAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/slideshow.png</normaloff>:/res/slideshow.png</iconset>
   </property>
   <property name="text">
    <string>Start slideshow</string>
   </property>
  </action>
  <action name="settingsAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/settings.png</normaloff>:/res/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="zoomInAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/zoom-in.png</normaloff>:/res/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="zoomOutAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/zoom-out.png</normaloff>:/res/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="zoomWindowAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/zoom-window.png</normaloff>:/res/zoom-window.png</iconset>
   </property>
   <property name="text">
    <string>Window size</string>
   </property>
  </action>
  <action name="zoomOriginalAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/zoom-original.png</normaloff>:/res/zoom-original.png</iconset>
   </property>
   <property name="text">
    <string>Original size</string>
   </property>
  </action>
  <action name="resizeitemsAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/resize-items.png</normaloff>:/res/resize-items.png</iconset>
   </property>
   <property name="text">
    <string>Resize items...</string>
   </property>
  </action>
  <action name="flipHorizontalAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/flip-horizontal.png</normaloff>:/res/flip-horizontal.png</iconset>
   </property>
   <property name="text">
    <string>Flip horizontal</string>
   </property>
  </action>
  <action name="flipVerticalAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/flip-vertical.png</normaloff>:/res/flip-vertical.png</iconset>
   </property>
   <property name="text">
    <string>Flip vertical</string>
   </property>
  </action>
  <action name="nextimageAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/next.png</normaloff>:/res/next.png</iconset>
   </property>
   <property name="text">
    <string>Next image</string>
   </property>
  </action>
  <action name="previmageAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/prev.png</normaloff>:/res/prev.png</iconset>
   </property>
   <property name="text">
    <string>Previous image</string>
   </property>
  </action>
  <action name="editorsNewAction">
   <property name="text">
    <string>Add editor</string>
   </property>
  </action>
  <action name="editorsManagerAction">
   <property name="text">
    <string>Editors manager</string>
   </property>
  </action>
  <action name="shareImageShackAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/imageshack.png</normaloff>:/res/imageshack.png</iconset>
   </property>
   <property name="text">
    <string>ImageShack.us</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
